src/NOTD.SC2Map/MapScript.galaxy

```diff
@@ -17369,7 +17388,11 @@ bool gt_InfestorAI_Func (bool testConds, bool runActions) {
 void gt_InfestorAI_Init () {

```

Comment:
Going to have to rework this. The UnitRefFromVariable does not work with arrays, it can't resolve them. What I would suggest is back this part out so I can merge the rest. In a separate pull, register the infestors with this trigger as they are created, as it is done with Marines with the other events you optimized.

Reply:
Interesting that this now works, maybe it is fixed for other events.